WordPress/Resources/AppStoreStrings.po

```diff
@@ -13,6 +13,11 @@ msgstr ""
 "Last-Translator: \n"
 "Language-Team: \n"
 
+#. translators: The application name in the Apple App Store. Please do not translate the brand name, only any companion text. Limit to 30 characters including spaces and punctuation!
```

Comment:
Is "Please do not translate the brand name, only any companion text" clear enough? _Even though one would hope that the fact that "WordPress" should not be translated should go without saying._

Reply:
```suggestion
#. translators: The application name in the Apple App Store. Please keep the brand name ('WordPress') verbatim. Limit to 30 characters including spaces and punctuation!
```